src/main/webapp/recommendation-map.html

```diff
@@ -49,10 +49,10 @@ <h3>Want to view a recommendation?</h3>
       <!-- The Recommendation Map -->
       <div class="map-container">
         <div class="map-category-section">
-          <button class="map-category-button" id="restaurants-button">Restaurants</button>
-          <button class="map-category-button" id="places-button">Places to Visit</button>
-          <button class="map-category-button" id="bars-and-clubs-button">Bars and Clubs</button>
-          <button class="map-category-button" id="study-button">Study Places</button>
+          <button class="map-category-button" id="restaurants-button" onclick="initMap('Restaurants')">Restaurants</button>
+          <button class="map-category-button" id="places-button" onclick="initMap('Places To Visit')">Places to Visit</button>
+          <button class="map-category-button" id="bars-and-clubs-button" onclick="initMap('Bars and Clubs')">Bars and Clubs</button>
+          <button class="map-category-button" id="study-button" onclick="initMap('Study Places')">Study Places</button>
```

Comment:
I really have no idea why this change is included here 🤔 It was added in https://github.com/googleinterns/step241-2020/commit/c1625c8ae4bfb10dae2d93ed3e01ce201445a21c

Reply:
Me neither!

Reply:
Interesting.. hopefully it will not cause any problem when you merge into master :)